test.js

```diff
@@ -337,7 +338,7 @@ async function testFileNodeFromStream(t, ext, name) {
 	t.is(typeof fileType.mime, 'string', 'fileType.mime');
 }
 
-async function loadEntireFile(readable) {
+async function loadEntireFileFromNodeReadable(readable) {
```

Comment:
You can just use https://github.com/sindresorhus/get-stream for this and `loadEntireFileFromWebStream`. No need for custom logic.